runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerLog.java

```diff
@@ -40,13 +43,29 @@
 import net.runelite.client.util.StackFormatter;
 
 @Getter
-class LootTrackerBox extends JPanel
+class LootTrackerLog extends JPanel
 {
 	private static final int ITEMS_PER_ROW = 5;
-	private final long totalPrice;
 
-	LootTrackerBox(final ItemManager itemManager, final String title, final String subTitle, final LootTrackerItemEntry[] items)
+	private final JPanel itemContainer = new JPanel();
+	private final JLabel priceLabel = new JLabel();
+	private final JLabel subTitleLabel = new JLabel();
+
+	private final ItemManager itemManager;
+
+	private long totalPrice;
+	private int totalKills;
+
+	@Getter
+	private String title;
+
+	private List<LootTrackerEntry> entries = new ArrayList<>();
+
+	LootTrackerLog(final ItemManager itemManager, LootTrackerEntry entry)
```

Comment:
Having both initial construction from entry and method addEntry seems weird.

Reply:
Because every log contains atleast one entry, the initial one, new ones can be added later